manticore/manticore.py

```diff
@@ -294,37 +268,6 @@ def maxstorage(self):
     def maxstorage(self, max_storage):
         self._maxstorage = max_storage
 
-    @property
-    def verbosity(self):
```

Comment:
As an example, I feel like https://github.com/trailofbits/manticore/pull/445/files#diff-1906be2ef8b0ab48f2d00bca1d684961R75 feels a little inconsistent with the two lines immediately after